Makefile

```diff
@@ -24,6 +24,8 @@ deps-update:
 	go mod tidy && \
 	go mod vendor
 
+test: unit-tests
+
```

Comment:
Why are the identical paths needed? It could be `test` only.

Reply:
Didn't want to break any existing tooling that uses `make unit-tests` somewhere.